src/TapRating.tsx

```diff
@@ -95,6 +95,11 @@ export type TapRatingProps = {
    * Pass in a custom base image source
    */
   starImage?: string;
+
+  /**
+   * Testing ID for Jest
+   */
+  testID?: string;
```

Comment:
let's make this camelCase. So this should be `testId`

Reply:
On it!